kickstart/pack/xqau52/hybris

```diff
@@ -100,36 +93,7 @@ dd if=home.img bs=$BLOCKSIZE count=$HOMEBLOCKS of=/dev/sailfish/home
 
 /usr/sbin/vgchange -a n sailfish
 
-pigz -7 root.img && md5sum -b root.img.gz > root.img.gz.md5&
```

Comment:
We don't have factory image?

Reply:
No, we don't have partition to flash it to. Factory images on Sailfish X should be used available flashing images from the website to reflash device.

Reply:
OK